benchmarks/src/postprocessing/serve.py

```diff
@@ -55,11 +62,104 @@ def get(self, key: str):
             source_code = html_file.read()
             self.write(source_code)
 
+    class CompareOverview(web.RequestHandler):
+        def get(self, key: str):
+            df = create_database_df(database)
+            key = key.split("(")[1].split(")")[0]
+            df = df[df['workload-params'] == key]
+            max_index = df['index'].max()
+            samples = [df[df['index'] == i] for i in range(max_index + 1)]
+            tables = [i['duration'].describe().to_frame() for i in samples]
+            compare1 = tables[0] > tables[1]
+            compare0 = tables[1] > tables[0]
+            color_0 = lambda x: ["background-color:red" if i[1][0] else "" for i in compare0.iterrows()]
+            color_1 = lambda x: ["background-color:red" if i[1][0] else "" for i in compare1.iterrows()]
+            # tables[1].style.apply_index(color_b)
+            output = ""
+            tables[0].style.set_uuid("table0")
+            tables[1].style.set_uuid("table1")
+            output += tables[0].style.apply(color_0).set_uuid("table0").to_html()
+            output += tables[1].style.apply(color_1).set_uuid("table1").to_html()
+
+            plt.scatter(df['index'], df["duration"], s=10)
+            plt.xticks(list(range(max_index + 1)))
+            plt.ylabel("Duration [ms]")
+            plt.savefig("img.png")
```

Comment:
This is a really hacky solution with storing the `img.png` file. The image should be either generated on demand or cached in memory.